README.md

```diff
@@ -69,6 +68,8 @@ Arguments:
   validation.
 * `primary_key` (optional): The primary key of the model. Used to sort unmatched
   results for row-by-row validation.
+* `summary` (optional): Allows you to switch between a summary or detaied view
+  of the compared data. Accepts `true` or `false` vaules. Defaults to `true`.
```

Comment:
```suggestion
* `summary` (optional): Allows you to switch between a summary or detailed view
  of the compared data. Accepts `true` or `false` values. Defaults to `true`.
```
Couple of typos here and below. 

Also, I would suggest `summarise` (or `summarize` 😬) as the argument name, so that it's a verb (as a noun, it sorta implies that you are meant to pass _in_ some sort of summary object)